pkg/config/server/server.go

```diff
@@ -97,6 +97,9 @@ type ConfigFileRuntime struct {
 	// GRPCMaxMsgSize is the maximum message size that the grpc server will accept
 	GRPCMaxMsgSize int `mapstructure:"grpcMaxMsgSize" json:"grpcMaxMsgSize,omitempty" default:"4194304"`
 
+	// GRPCRateLimit is the rate limit for the grpc server
```

Comment:
should clarify that this is a per-second, per-engine rate limit

Reply:
done